common/useFissionAuth.ts

```diff
@@ -0,0 +1,171 @@
+import { useEffect, useState } from 'react'
+
+import * as webnative from 'webnative'
+import * as webnativeFilecoin from 'webnative-filecoin';
+
+import * as C from '@common/constants';
+import * as R from '@common/requests';
+
+import Cookies from 'js-cookie';
+
+webnative.setup.debug({ enabled: true })
+
+export function useFissionAuth() {
+  const [state, setState] = useState<webnative.State>(null)
+  let fs;
+  let authScenario: webnative.Scenario | null = null;
+  let username: string = null;
+
+
+  /** Webnative Initialization
+   * Load webnative and configure permissions.
+   */
+
+  useEffect(() => {
+    async function getState() {
+      const result = await webnative.initialise({
+        permissions: {
+          // The Estuary token is stored in app storage
+          // at bgins/estuary-www.
+          app: {
+            name: 'estuary-www',
+            creator: 'bgins',
+          },
+          fs: {
+            // The cosigner key is stored in the private filesystem
+            // at the path Keychain/fil-cosigner.
+            private: [webnativeFilecoin.DEFAULT_KEY_PERMISSION]
```

Comment:
I'd recommend setting a non-default key path here just so we don't clobber anyone using our demo (or other apps)... `Keychain/estuary` or something along those lines.

Reply:
Good point! I'll change this over to `Keychain/estuary-fil-cosigner`.